packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -629,6 +630,51 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 		}
 	}
 
+	public get disposed() {
+		if (!(this._disposed || this.socket.connected)) {
```

Comment:
I'm not sure there is any value in this telemetry. In a sense that without stacks it's not very actionable, especially that we know that could happen.
I think you are trying to create some venue to learn when this new beahavior (of ignoring ops / signals) might result in a bug, but I'm not sure I see an easy way to add triggers to detect that. The only thing that comes to mind - scheduling a micro-task to validate that when it runs, connection object is disposed. Maybe we should do it? It feels more direct and will generate errors in telemetry that we will notice right away.